strategies.py

```diff
@@ -26,10 +26,10 @@
 
 import go
 
-MAX_GAME_DEPTH = int(go.N * go.N * 1.25)
 # When to do deterministic move selection.  ~30 moves on a 19x19, ~8 on 9x9
 TEMPERATURE_CUTOFF = int((go.N * go.N) / 12)
 
+MAX_DEPTH = (go.N ** 2) * 1.4  # ~500 moves for 19x19, ~110 for 9x9
```

Comment:
would be nice to cast this to int and give exact number in the comment, just in case somebody goes grepping for "why did my game end at move 505"

Reply:
Done